### DISCUSSION TITLE
css modules make it configurable instead returning as proxy

### DISCUSSION THREAD

**Discussion by aelbore** [vitest-dev/vitest#1970] [Q&A] [answered]
does it possible to make it configurable for css module instead of returning it as proxy?

**Comment by sheremet-va** [comment, marked as answer]
Yes, you can configure `classNameStrategy`: https://vitest.dev/config/#css-modules-classnamestrategy

You would still need to enable CSS for it to actually process `module`, using `css.include`: 

```ts
export default {
  test: {
    css: {
      include: [/\.module\./]
    }
  }
}
```

You don't have to specify CSS extension, because this include only applies to CSS files.
